File: SQL/Midterm files/Acme_Used_CarsWPFcs/Acme_Used_CarsWPFcs/CustomerDB.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Data;
using System.Data.SqlClient;
using System.Windows;

namespace Acme_Used_CarsWPFcs
{
    public static class CustomerDB
    {
        public static bool InsertCustomer(Customer customer)
        {
            int rows = 0;
            SqlConnection con = Acme_Used_CarsDB.GetConnection();
            SqlCommand cmd = new SqlCommand();
            cmd.CommandText = "insert into Customer values (@number,@name,@inventoryid)";
            cmd.CommandType = CommandType.Text;
            cmd.Connection = con;
            cmd.Parameters.AddWithValue("@number", customer.number);
            cmd.Parameters.AddWithValue("@name", customer.name);
            cmd.Parameters.AddWithValue("@inventoryid", customer.inventoryid);
            try
            {
                con.Open();
                rows = cmd.ExecuteNonQuery();
            }
            catch (SqlException err)
            {
                MessageBox.Show(err.Message, "Database Error");
            }
            finally
            {
                con.Close();
            }
            if (rows == 1)
            {
                return true;
            }
            else
            {
                return false;
            }
        }
    }
}

File: SQL/Midterm files/Acme_Used_CarsWPFvb/Acme_Used_CarsWPFvb/CustomerDB.vb
Imports System.Data
Imports System.Data.SqlClient

Public Class CustomerDB

    Public Shared Function InsertCustomer(customer1 As Customer) As Boolean
        Dim rows As Integer = 0
        Dim con As SqlConnection = Acme_Used_CarsDB.GetConnection()
        Dim cmd As New SqlCommand()
        cmd.CommandText = "insert into Customer values (@number,@name,@inventoryid)"
        cmd.CommandType = CommandType.Text
        cmd.Connection = con
        cmd.Parameters.AddWithValue("@number", customer1.Number)
        cmd.Parameters.AddWithValue("@name", customer1.Name)
        cmd.Parameters.AddWithValue("@inventoryid", customer1.InventoryID)
        Try
            con.Open()
            rows = cmd.ExecuteNonQuery()
        Catch err As SqlException
            MessageBox.Show(err.Message, "Database Error")
        Finally
            con.Close()
        End Try
        If rows = 1 Then
            Return True
        Else
            Return False
        End If
    End Function

End Class
